command/command.go

```diff
@@ -229,3 +237,62 @@ func computeNewlineTargets(before, after string) []int {
 
 	return result
 }
+
+// computeMedianIndent determines the median indent of the given string yaml content.
+func computeMedianIndent(yamlContent string) int {
+	lines := strings.Split(yamlContent, "\n")
+	allSpaces := []int{}
+	lastLine := 0
+	for i := 1; i < len(lines); i++ {
+		lineBefore := lines[lastLine]
+		lineAfter := lines[i]
+		// Skip empty lines
+		if strings.TrimSpace(lineBefore) == "" {
+			lastLine++
+			continue
+		}
+		// Do not compare empty lines.
+		if strings.TrimSpace(lineAfter) == "" {
+			continue
+		}
+		l0 := leadingSpaces(lineBefore)
+		l1 := leadingSpaces(lineAfter)
+		lastLine = i
+		// Skip no change in indent.
+		if l0 == l1 {
+			continue
+		}
+		allSpaces = append(allSpaces, absDiffInt(l1, l0))
```

Comment:
Doesn't this lead to having no weight?

I know it's the median, but if you have a file with 200 lines with 2 spaces, and one with 4 spaces.

What will your code suggest?